src/index.ts

```diff
@@ -287,17 +287,17 @@ function initMappers(options: TsResolverOptions) {
       ? options.project
       : [process.cwd()]
 
-  const ignore = ['./node_modules/**']
+  const ignore = ['!**/node_modules/**']
 
-  mappers = configPaths
-    // turn glob patterns into paths
-    .reduce<string[]>(
-      (paths, path) => [
-        ...paths,
-        ...(isGlob(path) ? globSync(path, { ignore }) : [path]),
-      ],
-      [],
-    )
+  // turn glob patterns into paths
+  const projectPaths = [
```

Comment:
Can you not change the logic?

Reply:
Never mind, it seems similar in ts-eslint.

Reply:
yep that's what i based it on